lib/checks/hasUnsupportedNodes.js

```diff
@@ -1,30 +1,29 @@
-const Node = require('../node');
+const {query} = require('math-nodes');
+
 const resolvesToConstant = require('./resolvesToConstant');
 
 function hasUnsupportedNodes(node) {
-  if (Node.Type.isParenthesis(node)) {
```

Comment:
Yup! If there are valid trees handled by math-parser that aren't handled in mathsteps we might want to keep something like this, but hopefully we can always just default to not doing anything when it's a node we don't recognize